src/utils/FormInputHandling.ts

```diff
@@ -42,4 +42,70 @@ async function getDataWithoutIds(currentPage: number, limit: number) {
   };
 }
 
-export { getData, getDataWithoutIds };
+async function updateData(id: number, updatedSessionData: Partial<RowType>) {
+  const { data } = await instance.patch<PatchTypes[]>(
+    `api/session/${id}`,
+    updatedSessionData
+  );
+
+  return {
+    data,
+  };
+}
+
+function formatDateForPicker(date: Date): string {
+  return date.toISOString().split("T")[0]; // Extracts the date part
+}
+
+function formatTimeForPicker(date: Date): string {
+  return date.toISOString().split("T")[1].slice(0, 5); // Extracts the time part
+}
+
+async function getASession(id: number) {
```

Comment:
i feel this is unnecessary. have the function of course. but don't use it yet. its not needed for edit i feel